### DISCUSSION TITLE
章节字体问题

### DISCUSSION THREAD

**Discussion by wucong1997** [sjtug/SJTUThesis#884] [提问]
我用的latex模板，目录中的章节标题比如：第一章 绪论，目前是黑体加粗的，怎么改成宋体加粗呀

**Comment by LogCreative** [comment]
导言区添加

```latex
\ctexset{
  chapter/format=\zihao{3}\bfseries\centering 
}
```

**Reply by wucong1997** [reply]
这样改好像只是改了正文中的标题格式，目录中的还是黑体，我现在只想改目录中的字体，有什么办法嘛

**Reply by wucong1997** [reply]
而且我现在的字体是fandolhei，怎么改成正宗的黑体字体呀

**Reply by wucong1997** [reply]
我现在基本解决了，把原先的fandolhei字体改成正宗黑体，主要修改了如下代码（使用的是SJTU在线平台）：
`% 载入 SJTUThesis 模版
\documentclass[type=master,oneside, fontset=windows]{sjtuthesis}`

**Reply by LogCreative** [reply]
目前最新版的目录就是宋体，如果你使用了 v1 版本，改为宋体需要在导言区添加

```latex
\renewcommand\cftchapfont{\bfseries}
```

以及最新版不再支持使用 `fontset` 设定中文字体，改用 `cjk-font=windows`
